domains/raphael.json

```diff
@@ -4,11 +4,6 @@
         "email": "raphaelas.pro@gmail.com"
     },
     "record": {
-        "A": [
-            "185.199.108.153",
-            "185.199.109.153",
-            "185.199.110.153",
-            "185.199.111.153"
-        ]
+        "CNAME": "raphael-as.github.io."
```

Comment:
remove the . from raphael-as.github.io

Reply:
sorry haha, i just copy past from the github suggested cname